config/config_test.go

```diff
@@ -0,0 +1,55 @@
+package config_test
+
+import (
+	"os"
+
+	"github.com/tidepool-org/platform/config"
+	"github.com/tidepool-org/platform/store"
+
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+)
+
+var _ = Describe("Config", func() {
+
+	Describe("FromJson", func() {
+
+		It("loads the given config file", func() {
+			var mgoConfig store.MongoConfig
```

Comment:
thats because I started doing the naming updates on the https://github.com/tidepool-org/platform/tree/data-validation branch and then when I got to the the errors branched off. But yeah it will change from being an abbreviation